content/en/docs/tasks/extend-kubernetes/custom-resources/custom-resource-definitions.md

```diff
@@ -1196,6 +1197,17 @@ openAPIV3Schema:
 This is because each item of `foo` is itself an array, and each subarray in turn calls `self.all`. Avoid nested
 lists and maps if possible where validation rules are used.
 
+#### Validation Rules Metrics
+
+Two metrics are provided for validation rules currently: `cel_compilation_duration_seconds` and `cel_evaluation_duration_seconds`. 
+Those metrics could be used to track the time costed in compilation or evaluation time.
+```
+# HELP cel_compilation_duration_seconds [ALPHA] 
+# TYPE cel_compilation_duration_seconds histogram
+# HELP cel_evaluation_duration_seconds [ALPHA] 
+# TYPE cel_evaluation_duration_seconds histogram
+```
```

Comment:
We only document stable metrics at the moment:
- https://kubernetes.io/docs/concepts/cluster-administration/system-metrics/

If we add a reference page that lists all metrics, including unstable metrics, then these are welcome to go into that list. I wouldn't publish them separately as the instrumentation folks have plenty of other areas of docs that they are I think prioritising.
Instead of listing these in the task page, how about mentioning them in the related blog article - see #35459

Reply:
Thank you! I have removed the metrics.